internal/pkg/cli/pipeline_init.go

```diff
@@ -270,19 +270,20 @@ func (o *initPipelineOpts) askRepository() error {
 }
 
 func (o *initPipelineOpts) askGitHubRepoDetails() error {
-	o.provider = ghProviderName
+	// If the user uses a flag (now hidden) to specify a GitHub access token,
```

Comment:
```suggestion
	// If the user uses a flag to specify a GitHub access token,
```
removing (now hidden) since in the future the now won't make much sense :D